sympy/physics/quantum/qexpr.py

```diff
@@ -129,10 +129,10 @@ def __new__(cls, *args, **old_assumptions):
         """
 
         # First compute args and call Expr.__new__ to create the instance
-        args = cls._eval_args(args)
+        args = cls._eval_args(args, **old_assumptions)
```

Comment:
It seems that `old_assumptions` was never used to set assumptions, or, for any other purpose, in this module. Maybe we could change the misleading name to something more neutral (e.g. `kwargs`).

Reply:
agree, I simply followed the existing argument name, but it must be better to use a neutral name.

Also, quick grep shows there are number of `old_assumptions` under physics/quantum.
I feel little uncomfortable with existing `old_assumptions` because they are just passed to `Expr.__new__`, but the current `Expr.__new__` doesn't seem to accept keyword args.

It might be out of scope of this PR, but is it OK to remove them too?

```
$ grep -n old_assumption sympy/physics/quantum/*.py
sympy/physics/quantum/operator.py:391:    def __new__(cls, *args, **old_assumptions):
sympy/physics/quantum/operator.py:421:            obj = Expr.__new__(cls, *(kets[0], bras[0]), **old_assumptions)
sympy/physics/quantum/operator.py:430:                                                 **old_assumptions))
sympy/physics/quantum/operator.py:434:                                             **old_assumptions))
sympy/physics/quantum/operator.py:438:                                             **old_assumptions))
sympy/physics/quantum/qexpr.py:103:    def __new__(cls, *args, **old_assumptions):
sympy/physics/quantum/qexpr.py:132:        args = cls._eval_args(args, **old_assumptions)
sympy/physics/quantum/qexpr.py:134:            args = cls._eval_args(tuple(cls.default_args()), **old_assumptions)
sympy/physics/quantum/qexpr.py:141:    def _new_rawargs(cls, hilbert_space, *args, **old_assumptions):
sympy/physics/quantum/qexpr.py:151:        obj = Expr.__new__(cls, *args, **old_assumptions)
```

Reply:
That is probably OK, but I would leave that to another PR.

Reply:
OK, then I will only change the name to `kwargs` here, and create another PR later for `old_assumptions`. Thank you for your help!